docs/core/integration-quality-scale/rules/unique-config-entry.md

```diff
@@ -17,6 +17,8 @@ This can lead to duplicated devices and entities with unique identifiers collidi
 Any discovery flow must also ensure that a config entry is uniquely identifiable, as otherwise, it would discover devices already set up.
 
 To prevent this, we need to ensure that the user can only set up a device or service once.
+Trying to set up a device or service that is already set up should be prevented by the integration.
+If an update to the configuration is required, it should be handled by the reconfiguration flow instead, as it allows us to be more consistent with what gets updated.
```

Comment:
Should this discuss reauth as the only exception to this?

Reply:
Hmm, the goal is to disallow starting a config flow manually and then trying to setup the same device and automatically update things like the host and port. Not necessarily to only allow the reconfigure flow to update the config. (like, we still want to update the entry via code if needed, so I don't want to slam that door closed)

Reply:
True, I don't intend to suggest that reauth should be updating configuration outside of authentication credentials.  I was thrown off by the phrase "update to the configuration"  which sounds broad, but i understand you're implicitly allowing an exception for authentication parts of the configuration.

Reply:
But you are a native English speaker, so if you have improvements to convey this better, please let me know 🙏🏻